authors/apps/authentication/helper_functions/send_email.py

```diff
@@ -0,0 +1,26 @@
+from django.contrib.sites.shortcuts import get_current_site
+from django.template.loader import render_to_string
+from django.core.mail import EmailMultiAlternatives
+from django.utils.html import strip_tags
+import os
```

Comment:
As a good practice, have the system imports at the top of the other imports.

Reply:
Sure thing